Snowflake.Data/Core/Authenticator/OktaAuthenticator.cs

```diff
@@ -46,11 +46,6 @@ async Task IAuthenticator.AuthenticateAsync(CancellationToken cancellationToken)
         {
             logger.Info("Okta Authentication");
 
-            // Clear cookies before authenticating because when a cookie is present in the request,
-            // Okta will assume it is coming from a browser and perform a CSRF check.
-            // This will ensure that we are NOT including the ‘sid’ cookie with the request.
-            HttpUtil.ClearCookies(oktaUrl);
```

Comment:
So we don't need to clear cookies any more?

Reply:
I disabled the cookies in the httpHandler (UseCookies = false // Disable cookies) so it should not be necessary anymore.
I'm pretty sure it tested that it was working fine, but I'll run the Okta test again to be 100% certain.

Reply:
Retested okta, external browser and JWT, it works fine. Turning off cookies in the http handler means that we don't need to clean-up.